### PR TITLE
feat: Multiple conversation

### LINKED ISSUE #1132: Feature: Display Online Status, response time on the widget
### Problem

In the current widget view, team availability is shown as a toast and there is no information about the time people will take to reply.

### Solution

- Remove the existing message input box and show an agent availability along with the response time.

### Acceptance criteria

- Add a DB field for the user to choose the response time
  - Replies in a few minutes
  - Replies in a few hours
  - Replies within a day

- Display the list of online agents and the response time text.
- Remove the input box for the message, introduce a button which says send a message.

### PR DESCRIPTION
**Pull request by pranavrajs**
Fixes #1132 

Screenshot 2020-09-07 at 11 05 14 PM

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue [hunk 1]
```diff
@@ -347,6 +372,8 @@ export default {
         this.channelWelcomeTitle = this.inbox.welcome_title;
         this.channelWelcomeTagline = this.inbox.welcome_tagline;
         this.selectedFeatureFlags = this.inbox.selected_feature_flags || [];
+        console.log(this.inbox);
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue:375]: Remove

File: app/javascript/widget/components/ChatHeaderExpanded.vue [hunk 2]
```diff
@@ -68,7 +70,7 @@ export default {
 
   .body {
     color: $color-body;
-    font-size: 1.8rem;
+    font-size: 1.125rem;
```
Comment (nithindavid) [inline app/javascript/widget/components/ChatHeaderExpanded.vue:73]: Variable ?

File: app/javascript/widget/views/Home.vue [hunk 3]
```diff
@@ -1,26 +1,72 @@
...
+        enter-to-class="opacity-100 transform translate-y-0"
+        leave-class="opacity-100 transform translate-y-0"
+        leave-to-class="opacity-0 transform translate-y-16"
+      >
+        <div v-if="showInputTextArea && isOnMessageView" class="input-wrap">
+          <chat-footer />
+        </div>
+      </transition>
+      <transition
+        enter-active-class="transition-all delay-300 duration-300 ease-out"
```
Comment (nithindavid) [inline app/javascript/widget/views/Home.vue:57]: Remove translation-all and target only the affecting properties, Like `transition-opacity transition-transform` if they are available, Also see if we can add `will-change` css property to these with `transform, opacity` to get butter smooth animations.

**Review by nithindavid — CHANGES_REQUESTED** [review]

File: app/javascript/widget/views/Home.vue (hunk 3 above)
Comment (nithindavid) [reply app/javascript/widget/views/Home.vue:57]: https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

Comment (pranavrajs) [reply app/javascript/widget/views/Home.vue:57]: I guess will-change needs to be added only if there are problems with transition performance. Abusing it is not recommended.

I will change the transition-all to transform and opacity

File: app/javascript/widget/components/ChatHeaderExpanded.vue (hunk 2 above)
Comment (pranavrajs) [reply app/javascript/widget/components/ChatHeaderExpanded.vue]: Moved to classes

**Comment by pranavrajs** [comment]
Test the following

- [ ]  Set Locale
- [ ]  Set User
- [ ]  Set Custom Attributes
- [ ]  Set Label / Remove Label should throw deprecation message
- [ ]  Unread view
- [ ]  Conversation Initiated URL
- [ ]  Chatwoot Branding referrer
- [ ]  Delete custom attribute
- [ ]  Widget invisible

**Comment by pranavrajs** [comment]
Closing as this branch cannot be recovered. I will create a new PR with the changes required.

**Pull request closed without merging**